lambda_functions/vetext_incoming_forwarder_lambda/vetext_incoming_forwarder_lambda.py

```diff
@@ -156,30 +176,50 @@ def make_vetext_request(request_body):
 
     json_data = json.dumps(body)
 
+    logger.info("Making POST Request to VeText using: " + os.getenv('vetext_api_endpoint_domain') + os.getenv('vetext_api_endpoint_path'))
+    logger.debug(json_data)
+    
     connection.request(
         'POST',
-        os.environ.get('vetext_api_endpoint_path'),
+        os.getenv('vetext_api_endpoint_path'),
         json_data,
         headers)
-
+    
     response = connection.getresponse()
+    
+    logger.info("VeText call complete with response: %d", response.status)
```

Comment:
```suggestion
    logger.info(f"VeText call complete with response: {response.status}")
```
No %s in python strings please unless necessary. Use f-strings.